gnd/src/main/java/com/google/android/gnd/model/feature/Feature.java

```diff
@@ -39,8 +52,12 @@
   @Nullable
   public abstract String getCaption();
 
+  @Nullable
   public abstract Point getPoint();
 
+  @Nullable
```

Comment:
If we define this in a subclass of `Feature`, this accessor can be `@NonNull.` Same for `getPoint`.

Reply:
Same as above